### ISSUE TITLE
Extending to iOS and Android?

### ISSUE THREAD

**Issue by Happypig375** [radiolondra/YAMP2#2]
Avalonia will be supporting mobile platforms. Will this control be extensible to these new platforms as well?

**Comment by radiolondra** [comment]
Not as it is actually. The Unofficial libraries work fine with Windows, Linux, MacOS and some Raspberry OSes only. Personally I don't have any intention to extend them to mobile (at least not immediately). For mobile and Web I actually suggest to use [MAUI](https://dotnet.microsoft.com/en-us/apps/maui).

**Comment by Happypig375** [comment]
But using MAUI will mean little to no code reuse from Avalonia.

**Comment by radiolondra** [comment]
Yes, obviously. Avalonia cross-platform is actually in version 11-preview1 which has a lot of issues/regressions, e.g. see [my post about one of the issues I found](https://github.com/AvaloniaUI/Avalonia/issues/9043).
I don't know how much time they'll need to release a stable, working version. So, if you need to have your projects working immediately, I suggest to read about MAUI. It's useless to start working on the Unofficial libs and on whatever else project while there are so many issues which (for sure) will be corrected, but actually they aren't.

**Comment by vynxc** [comment]
what about now
